### PR TITLE
added ability to supply specific No2FA PubKeyHash(fixes #ZSDK-66)

### PR DESCRIPTION
**Pull request by vyastrebovvareger**

### CODE REVIEW

File: tests/test_wallet.py [hunk 1]
```diff
@@ -348,6 +348,17 @@ async def test_get_tokens(self):
     async def test_is_signing_key_set(self):
         assert await self.wallet.is_signing_key_set()
 
+    async def test_toggle_2fa(self):
+        result = await self.wallet.enable_2fa()
+        self.assertTrue(result)
+
+        pub_key_hash = self.wallet.zk_signer.pubkey_hash_str()
+        result = await self.wallet.disable_2fa(pub_key_hash)
+        self.assertTrue(result)
```
Comment (StanislavBreadless) [inline tests/test_wallet.py:357]: I think that it would be much better to get the account state and see if it is No2FA

Comment (vyastrebovvareger) [reply tests/test_wallet.py:357]: I see. Yeah. Will do

**Comment by vyastrebovvareger** [comment]
I do not know what has happened but at the moment request of toggle_2fa returns
```{'success': True}``` but the account state is not changing on the server-side. Is it possible to figure out the problem on the server-side somehow?

**Comment by vyastrebovvareger** [comment]
Also. What kind of doc API update would you like to see?
Should it be just Wallet API update or something else?

**Comment by vyastrebovvareger** [comment]
I corrected the test case based on the server-side logic of the `toggle_2fa` method. The question about documentation updates is still actual.

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**